js/chat/chat.ts

```diff
@@ -280,6 +283,36 @@ class ChatContainer extends LightElement {
     return html``;
   }
 
+  connectedCallback(): void {
+    super.connectedCallback();
+
+    // We use a sentinel element that we place just above the shiny-chat-input. When it
+    // moves off-screen we know that the text area input is now floating, add shadow.
+    let sentinel = this.querySelector<HTMLElement>(CHAT_INPUT_SENTINEL_TAG);
+    if (!sentinel) {
+      sentinel = createElement(CHAT_INPUT_SENTINEL_TAG, {
```

Comment:
This is a small thing, but I'd prefer if `CHAT_INPUT_SENTINEL_TAG` were local to this scope (to make it more obvious that the tag doesn't need to be defined)

Reply:
Done in 3843753d1126e4a4eb4802748629e6233d0eeaa8